docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -0,0 +1,266 @@
+
+Upgrade Procedure
+=================
+
+We recommend installing the new release next to the old one, running both in parallel, and making sure it is working as expected before uninstalling the old version.
+
+Change to the directory you want to install the new Monitoring stack.
+Download the latest release:
+You can download the .zip or the .tar.
+
+Install 4.y (The new version)
+-----------------------------
+
+.. code-block:: bash
+
+                wget -L https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-4.y.zip
+                unzip scylla-monitoring-4.y.zip
+                cd scylla-monitoring-scylla-monitoring-4.y/
+
+Replace “y” with the new minor release number, for example, 4.0.0.zip
+
+Setting the server's files
+--------------------------
+
+Copy the ``scylla_servers.yml`` and ``scylla_manager_servers.yml`` from the version that is already installed.
+
+.. code-block:: bash
+
+                cp /path/to/monitoring/3.x/prometheus/scylla_servers.yml prometheus/
+                cp /path/to/monitoring/3.x/prometheus/scylla_manager_servers.yml.yml prometheus/
+
+Validate the new version is running the correct version
+-------------------------------------------------------
+
+Starting from Scylla-Monitoring version 2.2, you can run:
+
+.. code-block:: bash
+
+                ./start-all.sh --version
+
+To validate the Scylla-Monitoring version.
+
+
+Validate the version installed correctly
+----------------------------------------
+
+To validate that the Monitoring stack starts correctly, first in parallel to the current (3.x) stack.
+
+.. code-block:: bash
+
+                ./start-all.sh -p 9091 -g 3001 -m 9095
+
+Browse to ``http://{ip}:9091``
+And check the Grafana dashboard
+
+Note that we are using different port numbers for Grafana, Prometheus, and the Alertmanager.
+
+.. caution::
+
+   Important: do not use the local dir flag when testing!
+
+When you are satisfied with the data in the dashboard, you can shut down the containers.
+
+.. caution::
+
+   Important: Do not kill the 3.x version that is currently running.
+
+Killing the new 4.y Monitoring stack in testing mode
+----------------------------------------------------
+
+Use the following command to kill the containers:
+
+.. code-block:: bash
+
+                ./kill-all.sh -p 9091 -g 3001 -m 9095
+
+You can start and stop the new 3.y version while testing.
+
+Move to version 4.y (the new version)
+-------------------------------------
+
+Note: migrating will cause a few seconds of blackout in the system.
+
+We assume that you are using external volume to store the metrics data.
+
+Kill all containers
+^^^^^^^^^^^^^^^^^^^
+
+At this point you have two monitoring stacks running side by side, you should kill both before
+continuing.
+
+Kill the newer version that runs in testing mode by following the instructions on how to `Killing the new 4.y Monitoring stack in testing mode`_
```

Comment:
```suggestion
#. Kill the newer version that runs in testing mode by following the instructions on how to `Killing the new 4.y Monitoring stack in testing mode`_
```

Reply:
I'm adding #. to turn it into a numbered list for clarity.

Reply:
I couldn't make it a continues numbered list, after other changes I hope it's no longer needed

Reply:
I don't know why it wouldn't render. Each #. should be converted to a number. We can skip it in this document, but using a numbered list for a sequence of steps is a good practice :)